### DISCUSSION TITLE
Query parameters?

### DISCUSSION THREAD

**Discussion by gordonfroyle** [pawelsalawa/sqlitestudio#4526] [General]
Assume that R(A) denotes a table R with a single column A and it has a few rows.

If I execute queries like

SELECT * FROM R WHERE A = :value;
SELECT * FROM R WHERE A = @value;
SELECT * FROM R WHERE A = ?;

(Note value is just an arbitrary string)

then SQLite Studio pops up a window asking me to specify a value for the "query parameter".

Is this feature documented anywhere in the manual? (I have looked through twice)

I'd just like to know more about this feature because it might be useful (for teaching purposes).

**Comment by pawelsalawa** [comment]
https://sqlite.org/lang_expr.html#parameters

**Reply by pawelsalawa** [reply]
If you're asking about SQLiteStudio's manual - then no, it's not documented there. Though that manual is not anywhere near completed.

**Reply by gordonfroyle** [reply]
Thank you - I was indeed asking about the SQLiteStudio manual, because I wondered how this feature was meant to be used and hoped to find the answer there.

But as I have never had occasion to use prepared statements in SQLite, the reference to the SQLite manual was still useful, so thanks again.
